docs/requirements.txt

```diff
@@ -14,21 +14,18 @@ joblib
 lxml
 matplotlib
 numpy
-pip
 pykml>=0.2
 pyproj
 scikit-image
 scipy
-# pyaps dependencies
-cdsapi
-ecCodes
-netcdf4
-pygrib
-# pyhdf  # required by MERRA, which is currently not supported in pyaps3
-# pyresample dependencies
+pysolid
```

Comment:
`requirements.txt` is a file typically associated with `pip` install requirements, but it's not mentioned/used for `pip` in the [installation instructions](https://mintpy.readthedocs.io/en/latest/installation/) and only used for `conda` (including in the file header comments). 

Unfortunately, using it with `pip` will *do the wrong thing* for the `pysolid` dependency as there is [a different `pysolid` on PyPI](https://pypi.org/project/pysolid/), so I'd suggest renaming this to `spec-file.txt` as in the [`conda` docs](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) or `conda-spec.txt` to clarify it's for use with `conda`. 

(and `requirements4rtd.txt` could just be moved to `docs/requirements.txt` then)

Reply:
I agree that `pip` with `pysolid` is confusing. `requirements.txt` is the default file for dependencies, and I don't see this small confusion as big enough to change it to a less familiar name.

Reply:
`requirements.txt` is the default file for python/PyPI dependencies, *not* conda dependencies, but I agree it'd be rather confusing to change it for existing users, so no need.